src/platform-implementation-js/driver-common/sidebar/ContentPanelViewDriver.ts

```diff
@@ -20,47 +20,48 @@ export interface ContentPanelDescriptor {
 }
 
 class ContentPanelViewDriver {
-  _driver: Driver;
-  _stopper: Kefir.Observable<null, unknown>;
-  _eventStream = kefirBus<{ eventName: string }, unknown>();
-  _isActive: boolean = false;
-  // This is not the `id` property passed by the application, but a random
-  // unique identifier used to manage a specific instance.
-  _instanceId: string = `${Date.now()}-${Math.random()}`;
-  _sidebarId: string;
-  _isGlobal: boolean;
+  #driver: Driver;
+  #stopper: Kefir.Observable<null, unknown>;
+  #eventStream = kefirBus<{ eventName: 'activate' | 'deactivate' }, unknown>();
+  #isActive: boolean = false;
+  /**
+   * This is not the `id` property passed by the application, but a random
+   * unique identifier used to manage a specific instance.
+   * */
+  #instanceId: string = `${Date.now()}-${Math.random()}`;
+  #sidebarId: string;
+  #isGlobal: boolean;
 
   constructor(
     driver: Driver,
     descriptor: Kefir.Observable<ContentPanelDescriptor, unknown>,
     sidebarId: string,
     isGlobal?: boolean,
   ) {
-    this._driver = driver;
-    this._sidebarId = sidebarId;
-    this._isGlobal = Boolean(isGlobal);
-    this._stopper = this._eventStream
+    this.#driver = driver;
+    this.#sidebarId = sidebarId;
+    this.#isGlobal = Boolean(isGlobal);
+    this.#stopper = this.#eventStream
       .ignoreValues()
       .beforeEnd(() => null)
       .toProperty();
-    const document = global.document; //fix for unit test
```

Comment:
This line was removed.

Reply:
It's a bit unclear what this means. Does this removal mean unit tests just work without it or we aren't referencing the file in tests?

Reply:
Unit tests work without this line. I suspect it was probably to account for an old version of `jsdom` or maybe even the pre-jest SDK test library (mocha?).